src/mm/memory_manager.rs

```diff
@@ -1,3 +1,16 @@
+//! VM-to-MMTk interface: safe Rust APIs.
+//!
+//! This module provides a safe Rust API for mmtk-core.
+//! We expect the VM binding to inherit and extend this API by:
+//! 1. adding their VM-specific functions
+//! 2. exposing the functions to native if necessary. And the VM binding needs to manage the unsafety
```

Comment:
Maybe rewrite as?:
exposing the functions to native if necessary (The VM binding needs to manage the unsafety for exposing this safe API to FFI).

Reply:
Sure. I will address your comments all the together when you finish the review.

Reply:
Yeah, sounds good. Feel free to do so. But probably be cautious about the time spent on this. I expect the open PRs will be merged today, and I plan to do another PR to rename `enum Allocator` to `AllocationSemantics` (and get it merged).